synapse/lib/stormhttp.py

```diff
@@ -331,7 +331,12 @@ def getObjLocals(self):
         }
 
     async def _httpRespJson(self):
-        return json.loads(self.valu.get('body'))
+        try:
+            return json.loads(self.valu.get('body'))
+        except json.JSONDecodeError as e:
+            mesg = f'Unable to decode HTTP response as json: {e.args[0]}'
+            logger.exception(mesg)
```

Comment:
I'm not a big fan of logging handled exceptions to stderr when they're not system events... feel strongly about this line belonging here?

Reply:
no strong attachments. removed.